modules/repository/branch.go

```diff
@@ -77,19 +77,19 @@ func SyncRepoBranchesWithRepo(ctx context.Context, repo *repo_model.Repository,
 				RepoID:        repo.ID,
 				Name:          branch,
 				CommitID:      commit.ID.String(),
-				CommitMessage: commit.CommitMessage,
```

Comment:
The meaning of "CommitMessage" in the branch struct has changed now.

It's in dev branch so there is still a chance to rename the field.

At least, there should be comment to say that it only contains the message summary but not the full message.